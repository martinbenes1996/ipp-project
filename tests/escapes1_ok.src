.IPPcode18

WRITE string@Ahoj!\032Jak\032se\032mas,\032chlape?
