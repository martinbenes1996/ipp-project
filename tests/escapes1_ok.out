Ahoj! Jak se mas, chlape?
